src/go/rpk/pkg/cli/cmd/redpanda/admin/brokers/brokers.go

```diff
@@ -70,9 +70,10 @@ func newListCommand(fs afero.Fs) *cobra.Command {
 					break
 				}
 			}
+			fmt.Printf("Cluster view version of the broker handling this request: %d\n", cv.Version)
```

Comment:
If I saw this message as the output, I'd immediately thought which broker was handling this request, if there is a way to ask another broker and how to get the most up to date information (at the moment of the request). Not sure if we should increase the scope of this PR (the last two questions) but we may include the broker we asked in the output.

Something like:

```
Querying broker XXX
Cluster view version: YYY
Brokers:
  - AAA
  - BBB
  - CCC
```

However since changing the rpk output is a customer visible it makes sense to get @twmb input on the user's expectations before to proceed.

Reply:
We'd need to change `sendAny` to return the broker that was queried. If so, the output could be something like,
```
From broker %d on cluster view version %d:\n
TA BL E
.. .. .
```

wdyt?

Reply:
looks good to me